dali/operators/reader/webdataset_reader_op.cc

```diff
@@ -123,6 +123,14 @@ with a semicolon (';') and may contain dots.
 
 Example: "left.png;right.jpg")code",
             DALI_STRING_VEC)
+    .AddOptionalArg("case_insensitive_extensions",
+      R"code(Determines whether the extensions provided via the `ext` should be case insensitive.
+
+Allows mixing case sizes in the `ext` argument as well as in the webdataset container. For example
+when turned on: jpg, JPG, jPG should work.
+
+If the extension characters cannot be represented as ASCI the result of this option is undefined.
+)code", false)
```

Comment:
Isn't the default `case_insensitive_extensions=false` a double negation? We say that, "we are not case insensitive by default". Wouldn't the "we are case senstivie by default" message be clearer?

As as +1 for the positive variant of the name is that we already have `case_sensitive_filter` param in file reader.

Reply:
Done